coax/_core/stochastic_q.py

```diff
@@ -48,15 +48,22 @@ class StochasticQ(BaseStochasticFuncType1):
 
         The gym-style environment. This is used to validate the input/output structure of ``func``.
 
-    value_range : tuple of floats
+    value_range : tuple of floats, optional
 
-        A pair of floats :code:`(min_value, max_value)`.
+        A pair of floats :code:`(min_value, max_value)`. If no `value_range` is given, `num_bins`
+        is the number of bins of the quantile function as in
+        `IQN <https://arxiv.org/abs/1806.06923>` or `QR-DQN <https://arxiv.org/abs/1710.10044>`.
 
     num_bins : int, optional
 
-        The space of rewards is discretized in :code:`num_bins` equal sized bins. We use the default
-        setting of 51 as suggested in the `Distributional RL <https://arxiv.org/abs/1707.06887>`_
-        paper.
+        If `value_range` is given: The space of rewards is discretized in :code:`num_bins` equal
+        sized bins. We use the default setting of 51 as suggested in the
+        `Distributional RL <https://arxiv.org/abs/1707.06887>`_paper.
+
+        Else: The number of fractions of the quantile function of the rewards is defined by
```

Comment:
Thanks for expanding the docstring as well.

Reply:
I am glad I can help and get some experience with jax!